packages/eas-cli/src/user/SessionManager.ts

```diff
@@ -147,10 +148,13 @@ export default class SessionManager {
   public async showLoginPromptAsync({
     nonInteractive = false,
     printNewLine = false,
+    sso = false,
   } = {}): Promise<void> {
     if (nonInteractive) {
       Errors.error(
-        `Either log in with ${chalk.bold('eas login')} or set the ${chalk.bold(
+        `Either log in with ${chalk.bold('eas login')} or ${chalk.bold(
+          'eas login --sso | -s'
+        )} or set the ${chalk.bold(
```

Comment:
revert since this in `non-interactive`, and SSO login in impossible in non-interactive.

Reply:
Done